### PR TITLE
Multiple axes support.

### LINKED ISSUE #24: plot call kwargs format
How do we ask the user to provide input values for the plot calls?  That is, specifically, the __call__ function inside the plotting class (e.g. Contourf), which can be called either by the user to produce a static plot or repeatedly by the Browser to produce a new plot for every slider movement.

Currently this is:
```python
def __call__(self, **kwargs)
```
but this may become confusing for anonymous dimensions, specifically for the case in which the user wants a static plot, so do we replace **kwargs with a list of tuples, or a dictionary, or something else?

(My preference is a dictionary).

### PR DESCRIPTION
**Pull request by bjlittle**
Building on prior PRs this is the final part in the re-architecture in order to support multiple axes

To do:
- [x] Rebase

Closes #24, #25, #34

### CODE REVIEW

**Comment by bjlittle** [comment]
@marqh Thar she blows! :whale:

File: lib/cube_browser/__init__.py [hunk 1]
```diff
@@ -14,13 +16,50 @@
...
 
+
 # Set default IPython magics if an IPython session has invoked the import.
 ipynb = IPython.get_ipython()
 if ipynb is not None:
     ipynb.magic(u"%matplotlib notebook")
     ipynb.magic(u"%autosave 0")
 
 
+class AxisAlias(namedtuple('AxisAlias', 'dim, name, size')):
```
Comment (marqh) [inline lib/cube_browser/__init__.py]: _AxisAlias

File: lib/cube_browser/__init__.py [hunk 2]
```diff
@@ -14,13 +16,50 @@
...
+        return result
+
+    def __ne__(self, other):
+        result = self.__eq__(other)
+        if result is not NotImplemented:
+            result = not result
+        return result
+
+
+class AxisDefn(namedtuple('AxisDefn', 'dim, name, size, coord')):
```
Comment (marqh) [inline lib/cube_browser/__init__.py]: _AxisDefn

File: lib/cube_browser/__init__.py [hunk 3]
```diff
@@ -237,43 +285,130 @@ def cache(self, value):
...
+        self.subcube = self.cache.setdefault(key, self.cube[index])
+        return self.draw(self.subcube)
+
+    def draw(self, cube):
+        """Abstract method."""
+        emsg = '{!r} requires a draw method for rendering.'
+        raise NotImplementedError(emsg.format(type(self).__name__))
+
+
+###############################################################################
```
Comment (marqh) [inline lib/cube_browser/__init__.py]: remove please

**Pull request merged**